mpas_analysis/ocean/ocean_modelvsobs.py

```diff
@@ -133,6 +133,35 @@ def ocn_modelvsobs(config, field, streamMap=None, variableMap=None):
         fileOutLabel = "sstHADOI"
         unitsLabel = r'$^o$C'
 
+    elif field == 'sss':
+
+        selvals = {'nVertLevels': 0}
+
+        obs_filename = "{}/Aquarius_V3_SSS_Monthly.nc".format(obsdir)
+        dsData = xr.open_mfdataset(obs_filename)
+
+        time_start = datetime.datetime(2011, 8, 1)
+        time_end = datetime.datetime(2014, 12, 31)
+
+        ds_tslice = dsData.sel(time=slice(time_start, time_end))
+
+        # The following line converts from DASK to numpy to supress an odd
+        # warning that doesn't influence the figure output
+        ds_tslice.SSS.values
```

Comment:
It would be good to submit an xarray / dask bug report if we could get a simple xarray or dask - only reproducible manifestation of this error if possible.  However, this may be time consuming and not worth it, at least in the short-term.